apps/services/sessions/src/app/sessions/sessions-query.dto.ts

```diff
@@ -1,10 +1,21 @@
 import { ApiPropertyOptional } from '@nestjs/swagger'
-import { IsOptional } from 'class-validator'
+import { IsDateString, IsIn, IsOptional } from 'class-validator'
 
 import { PaginationDto } from '@island.is/nest/pagination'
 
 export class SessionsQueryDto extends PaginationDto {
   @IsOptional()
+  @IsDateString()
+  @ApiPropertyOptional()
+  from?: Date
+
+  @IsOptional()
+  @IsDateString()
+  @ApiPropertyOptional()
+  to?: Date
+
+  @IsOptional()
+  @IsIn(['ASC', 'DESC'])
```

Comment:
In #9732 I actually changed `order` to be enum instead.
https://github.com/island-is/island.is/pull/9732/files#diff-094641991d893eee558676ee033d2385554275b715cc0c1797468915fa49c1cfR15